webrtc-stats.html

```diff
@@ -1340,7 +1339,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the RTT computed by the STUN connectivity checks [[!STUN-PATH-CHAR]].
+                  Represents the RTT computed by the STUN connectivity checks
+                  [[!STUN-PATH-CHAR]] as well as consent checks [[!RFC7675]].
```

Comment:
Is this an average, i.e., sum of the RTTs computed divided by the number of checks (connectivity and consent)?